style.css

```diff
@@ -113,766 +168,588 @@ a {
-    font-size: 20px;
-    color: rgb(96, 112, 255);
-    line-height: 60px;
-    text-align: start;
-    justify-content: center;
-    align-items: center;
-  }
+.partation {
+  display: flex;
+  flex-direction: row;
+}
 
-  .div-text {
-    display: flex;
-    position: absolute;
-    flex-direction: column;
-    text-align: justify;
-    justify-content: center;
-    font-family: Poppins, sans-serif;
-    font-weight: normal;
-    margin-left: 200px;
-    margin-right: 200px;
-    margin-top: 200px;
-    width: 50%;
-    padding-top: 30px;
-  }
-
-  .works {
-    display: grid;
-    justify-content: center;
-    align-items: center;
-    margin: 30px;
-    row-gap: 30px;
+@media screen and (min-width: 768px) {
+  .body {
+    background-repeat: no-repeat;
+    background-size: cover;
   }
 
-  .sections {
-    display: flex;
-    flex-direction: row;
-    column-gap: 50px;
+  .body > div {
+    margin-left: 260px;
+    margin-right: 260px;
   }
 
-  .text h2 {
-    font-family: Poppins, sans-serif;
-    font-weight: bold;
-    font-size: 40px;
-    color: rgb(23, 43, 77);
-    line-height: 40px;
-    display: flex;
-    margin-top: 40px;
-    margin-bottom: -5px;
+  .body > h3 {
+    margin-left: 260px;
   }
 
-  .tonic {
-    display: none;
+  .main {
+    width: 100%;
   }
 
-  .face {
-    display: block;
+  .navbar {
+    display: flex;
+    justify-content: space-around;
   }
 
-  .br {
+  #nav1 {
     display: none;
   }
 
-  .list ul {
+  #nav2 {
     display: flex;
-    margin-left: -40px;
-    align-items: flex-start;
   }
 
-  .CANOPY {
-    color: rgb(52, 69, 99);
-  }
-
-  .text p {
-    font-family: Poppins, sans-serif;
-    font-weight: normal;
-    font-size: 17px;
-    line-height: 24px;
-    color: rgb(52, 69, 99);
-    display: flex;
-    justify-content: flex-start;
-    width: 500px;
+  #nav2 > a {
+    text-decoration: none;
   }
 
-  .button-list ul {
-    display: flex;
-    margin-left: -40px;
+  .bar1,
+  .bar2,
+  .bar3 {
+    border: none;
+    font-size: 15px;
+    background-color: white;
+    color: black;
+    width: auto;
+    height: auto;
+    font-weight: lighter;
+    margin-left: 32px;
   }
 
-  .button-list ul li {
-    color: rgb(96, 112, 255);
-    padding: 10px;
-    border-radius: 6px;
-    background-color: rgb(235, 235, 255);
-    margin-right: 13px;
+  .header {
+    margin-right: 154px;
   }
 
-  .ruby {
-    display: block;
+  .project {
+    padding: 40px 142px 40px 142px;
   }
 
-  button {
-    padding: 10px 20px;
-    text-align: center;
-    text-decoration: none;
-    font-weight: bold;
-    font-size: 16px;
-    margin: 4px 2px;
-    cursor: pointer;
+  .profile_snapshoot {
+    margin-left: 0;
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+    border-radius: 16px;
   }
 
-  .btn:hover {
-    background-color: white;
-    border: 1.6px solid rgb(96, 112, 255);
-    border-radius: 8px;
-    color: rgb(96, 112, 255);
+  .project1 {
+    display: flex;
   }
```

Comment:
Great job @SaeqaSultani on implementing the changes.

- [ ] As I pointed out in the previous review, It is stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), you must use a JavaScript array to store all of the information for all projects. For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, and link to the source.
- [ ] Kindly fix the text color and font family as I stated in the previous review so that your design can look very similar to what is provided in the Figma template.
![Your popup](https://user-images.githubusercontent.com/63917725/181741107-2f7d3cf8-8c2f-4d99-8c09-b05d85254355.PNG)